### PR TITLE
Spark indexer suggest

### PR DESCRIPTION
**Pull request by LeonardoGonzales**

### CODE REVIEW

**Comment by jluo321** [comment]
databaseSuggestions() needs to filter out implicit database

**Comment by jluo321** [comment]
getKeyword(), you don't need to use flatfile. use KeywordRDDReader is enough. all keywords used by entries are in the keyword list

**Comment by jluo321** [comment]
The same applies to subcellular location

**Comment by jluo321** [comment]
others looks good

File: spark-indexer/src/main/java/org/uniprot/store/spark/indexer/suggest/mapper/flatfile/FlatFileToCofactorChebi.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import java.util.List;
+
+import org.apache.spark.api.java.function.PairFlatMapFunction;
+import org.uniprot.core.uniprot.comment.*;
+import org.uniprot.core.util.Utils;
+import org.uniprot.store.spark.indexer.suggest.SuggesterUtil;
+
+import scala.Tuple2;
+
+/**
```
Comment (eddturner) [inline spark-indexer/src/main/java/org/uniprot/store/spark/indexer/suggest/mapper/flatfile/FlatFileToCofactorChebi.java]: I think a short javadoc description of the mapping would be useful for newcomers.

Comment (eddturner) [reply spark-indexer/src/main/java/org/uniprot/store/spark/indexer/suggest/mapper/flatfile/FlatFileToCofactorChebi.java]: ... and the same for lots of the mapping classes. Sorry!

**Pull request merged**